zeroconf/__init__.py

```diff
@@ -1045,11 +1054,11 @@ def write_question(self, question: DNSQuestion) -> None:
         self.write_short(question.type)
         self.write_short(question.class_)
 
-    def write_record(self, record: DNSRecord, now: float) -> int:
+    def write_record(self, record: DNSRecord, now: float, allow_long: bool = False) -> bool:
         """Writes a record (answer, authoritative answer, additional) to
-        the packet"""
+        the packet.  Returns True on success. """
```

Comment:
Can you mention the failure scenarios in this docstring as well?

Reply:
Done.